### PR TITLE
New debug section for Get Started with Native Mobile

### PR DESCRIPTION
**Pull request by ConnorLand**
This PR contains a new **3.4 Debugging Your Implementation** section for the how-to _Get Started with Native Mobile._

This section gives generic instructions on how to start the debugging process. It does not explain how to debug, as that varies from app to app and is something any debugger (read: advanced developer) will know how to do independently.

### CODE REVIEW

**Comment by ConnorLand** [comment]
I want to be sure the user understands which thing (Mendix Studio Pro, their locally-running Mendix app, or their mobile Make It Native app) they must be using during each instruction. Has this been presented clearly enough in the current draft?

File: content/howto/mobile/getting-started-with-native-mobile.md [hunk 1]
```diff
@@ -127,9 +127,40 @@ You may notice an **Enable dev mode** toggle on the Make It Native app home page
 
 {{% image_container width="500" %}}![enable dev mode](attachments/getting-started-with-native-mobile/enable-dev-mode.png){{% /image_container %}}
 	
-{{% todo %}}[link to future debug doc - For more information on debugging, see Debugging A Native Mobile App]{{% /todo %}}
+### 3.4 Debugging Your Implementation
 
-### 3.4 Viewing Changes to Your App on Your Testing Device {#viewingchanges}
+When changing your native app or designing a custom widget, you may need to debug your implementation. The Make It Native app exposes a developer mode which supports debugging native apps for expert developers. Chrome is recommended for this, as it starts automatically during debugging.
```
Comment (Adam-Dupaski) [inline content/howto/mobile/getting-started-with-native-mobile.md]: I would say "Using Google Chrome is..."

File: content/howto/mobile/getting-started-with-native-mobile.md [hunk 2]
```diff
@@ -127,9 +127,40 @@ You may notice an **Enable dev mode** toggle on the Make It Native app home page
...
+
+To start a debugging session, do the following:
+
+1. Run your Mendix project locally on your desktop.
```
Comment (Adam-Dupaski) [inline content/howto/mobile/getting-started-with-native-mobile.md]: "app project" - never just "project"

File: content/howto/mobile/getting-started-with-native-mobile.md [hunk 3]
```diff
@@ -127,9 +127,40 @@ You may notice an **Enable dev mode** toggle on the Make It Native app home page
...
+2. Start the Make It Native app.
+3. Select **Enable dev mode** in the Make It Native app.
+4.  Start your project on your mobile device: </br>
```
Comment (Adam-Dupaski) [inline content/howto/mobile/getting-started-with-native-mobile.md]: You probably don't need 4, a, b - a should be 4, and b should be 5?

File: content/howto/mobile/getting-started-with-native-mobile.md [hunk 4]
```diff
@@ -127,9 +127,40 @@ You may notice an **Enable dev mode** toggle on the Make It Native app home page
...
+	a. In Mendix Studio Pro, click **View** > **View in the Mendix App**.</br>
+	b. With your mobile device, tap **Scan QR code**, then scan the QR code on your desktop.
+When the Make It Native app finishes loading your project, do the following:
+
+1. Open the developer menu by using a three-finger long press.
```
Comment (Adam-Dupaski) [inline content/howto/mobile/getting-started-with-native-mobile.md]: "three-finger long press" - you really need 3 fingers to open the menu?

**Comment by Adam-Dupaski** [comment]
> 
> 
> I want to be sure the user understands which thing (Mendix Studio Pro, their locally-running Mendix app, or their mobile Make It Native app) they must be using during each instruction. Has this been presented clearly enough in the current draft?

Yes I think that is all clear.

File: content/howto/mobile/getting-started-with-native-mobile.md (hunk 4 above)
Comment (ConnorLand) [reply content/howto/mobile/getting-started-with-native-mobile.md]: Yes, you need three fingers tapping together. Mobile shortcuts are weird.

File: content/howto/mobile/getting-started-with-native-mobile.md (hunk 2 above)
Comment (ConnorLand) [reply content/howto/mobile/getting-started-with-native-mobile.md]: Changed throughout

File: content/howto/mobile/getting-started-with-native-mobile.md (hunk 3 above)
Comment (ConnorLand) [reply content/howto/mobile/getting-started-with-native-mobile.md]: Changed

File: content/howto/mobile/getting-started-with-native-mobile.md (hunk 1 above)
Comment (ConnorLand) [reply content/howto/mobile/getting-started-with-native-mobile.md]: Changed

**Pull request merged**